toptal/picasso
# fix: remove tooltip blur delay

### Description

As much as we wanted to mimic the native browser behavior for the `title` attribute, which has a delay on hover and fades out on blur (observed Chrome), the on blur delay felt string with the tooltip, so how we only delay on hover. 

### How to test

On the Temploy, go to the tooltip examples page, hover and leave tooltips and observe that it has a delay on hover, but does NOT delay on blur. You'll also note that after the first hover on a given tooltip, the tooltip will appear immediately.

### Review
